cmd/snap/cmd_quota_test.go

```diff
@@ -286,21 +289,21 @@ func (s *quotaSuite) TestSetQuotaCpuHappy(c *check.C) {
 		cpuCount:      2,
 		cpuPercentage: 50,
 	}
+	// this data is not tested against, but it should still be valid
 	const getJsonTemplate = `{
 		"type": "sync",
 		"status-code": 200,
 		"result": {
 			"group-name":"foo",
-			"constraints": { "memory": %d },
-			"current": { "memory": 500 }
+			"constraints": { "cpu":{"count":%d, "percentage":%d} },
 		}
 	}`
```

Comment:
Why have this as a template if it's only used once? The new test has the same template and parameters so it could be shared. If the idea is to have it repeated next to each test code for clarity, then the values can just be in the JSON from the start IMO.

Reply:
Thanks for the review! And good point, there is no need for it to be a template, I'll change this in a followup